DataManagementSystem/DB/FileCatalogComponents/DirectoryMetadata/DirectoryMetadata.py

```diff
@@ -28,22 +30,25 @@ def setDatabase(self, database):
   def addMetadataField(self, pname, ptype, credDict):
     """ Add a new metadata parameter to the Metadata Database.
         pname - parameter name, ptype - parameter type in the MySQL notation
+        Modified to use fully qualified metadata names.
```

Comment:
probably better to update the doc string. Please use
:param str pname: ....

Reply:
I updated the docstrigs in the methods I modified to my best knowledge. Since I haven't written the code in the first place, some params description might be incomplete or even incorrect.